streams/src/main/java/org/apache/kafka/streams/processor/internals/RecoverableClientException.java

```diff
@@ -1,33 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
We do not need this class anymore since it is now merged into TaskMigratedException.